commcare_connect/opportunity/api/views.py

```diff
@@ -61,8 +60,7 @@ def get_queryset(self):
 
 
 class ClaimOpportunityView(APIView):
-    authentication_classes = [OAuth2Authentication]
-    permission_classes = [TokenHasReadWriteScope]
+    permission_classes = [IsAuthenticated]
```

Comment:
why did this need to change?

Reply:
The authentication and permission here were setup for client_credentials workflow. Since we have ConnectID as a auth also we can directly use the IsAuthenticated permission as we do for other apis.